tests/ESPresense.Companion.Simulation/AccuracyHarness/Csi/test_csi_replay.py

```
@@ -0,0 +1,132 @@
+
+class TestCorpusIntegrity(unittest.TestCase):
+    def test_corpus_present(self):
+        files = corpus_files()
+        self.assertEqual(len(files), 24,
+                         "expected the 24-file synthetic corpus (2 boards x 6 rates x 2 scenes)")
+
+    def test_every_file_validates(self):
+        """Every corpus file parses under the independent ecf1 container validator,
+        and each CSI record's payload matches the header-declared subcarrier count."""
+        for p in corpus_files():
+            h, counts, stat = ecf1.validate(p)
```

Ruff (0.15.15)

[warning] Unpacked variable `stat` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)